### DISCUSSION TITLE
Sonoff s31 partition wizard

### DISCUSSION THREAD

**Discussion by tismofied** [arendst/Tasmota#22181] [Q&A] [answered]
Please someone put my mind at ease. l I have had these sonoff s31 plugs for a couple of years now and i was wondering if I am supposed to have the partition wizard on them? because I don't have the tools option in the GUI
Also, what raised this question in my head is I am able to upgrade to 14.2.0 firmware.
Thank you so much!

**Comment by sfromis** [comment, marked as answer]
No, the Partition Wizard is only relevant for devices using ESP32, while the Sonoff S31 is using the older ESP8266. Hence upgrading Tasmota firmware has no dependencies here, and you should be able to just use normal OTA upgrade via an URL pointing to the new binary on the OTA web site.

**Reply by tismofied** [reply]
> No, the Partition Wizard is only relevant for devices using ESP32, while the Sonoff S31 is using the older ESP8266. Hence upgrading Tasmota firmware has no dependencies here, and you should be able to just use normal OTA upgrade via an URL pointing to the new binary on the OTA web site.

Thank you very much.
